lib/PropertyDetailsView.js

```diff
@@ -8,12 +8,16 @@ module.exports = function PropertyDetailsView(property) {
             <h6>Name</h6>
             <p>${property.name}</p>
             <h6>Type</h6>
-            <p>${property.return.resolvedType}</p>
+            <p>
+                ${_.map(property.types, (type) => `
+                    ${type.resolvedType}
+                `).join(`, `)}
+            </p>
```

Comment:
String interpolation is not needed here. Also, lodash has shorthand methods for mapping properties. So this could just become:
```
${_.map(property.types, 'resolvedType').join(', ')}
```